test/blackbox-tests/test-cases/coq/vos-build.t/dune-vo

```diff
@@ -0,0 +1,9 @@
+(coq.theory
+ (name basic)
```

Comment:
Add (mode vo) here

Reply:
Dune will autodetect coq-native which is enabled in the test suite. You can force the mode by giving the field

Reply:
OK thanks. Done.